ui/kotlinx-coroutines-javafx/test/JavaFxStressTest.kt

```diff
@@ -0,0 +1,39 @@
+package kotlinx.coroutines.javafx
+
+import javafx.beans.property.SimpleIntegerProperty
+import kotlinx.coroutines.*
+import kotlinx.coroutines.flow.first
+import org.junit.Before
+import org.junit.Test
+
+class JavaFxStressTest : TestBase() {
+
+    @Before
+    fun setup() {
+        ignoreLostThreads("JavaFX Application Thread", "Thread-", "QuantumRenderer-", "InvokeLaterDispatcher")
+    }
+
+    @Test
+    fun cancellationRaceStressTest() = runTest {
```

Comment:
`testCancellationRace`: `test` should be in the prefix and `stressTest` is already present in the class name

Reply:
Fixed.